docs/nodejs/vuejs-tutorial.md

```diff
@@ -9,133 +9,127 @@ MetaDescription: Vue JavaScript tutorial showing IntelliSense, debugging, and co
+The Volar extension supplies Vue.js language features (syntax highlighting, IntelliSense, formatting) to VS Code. The TypeScript Vue Plugin is not necessary for this tutorial.
 
-![vetur extension](images/vuejs/vetur-extension.png)
+![Volar extension](./images/vuejs/volar-extension.png)
 
-From the notification, press **Install** to download and install the Vetur extension. You should see the Vetur extension **Installing** in the Extensions view. Once the installation is complete (may take several minutes), the **Install** button will change to the **Manage** gear button.
+From the notification, press **Install** to download and install the Volar extension.
 
 Now you should see that `.vue` is a recognized file type for the Vue language and you have language features such as syntax highlighting, bracket matching, and hover descriptions.
 
-![vue language features](images/vuejs/vue-language-features.png)
+![Volar syntax highlighting](./images/vuejs/syntax-highlighting.png)
+
+You can learn more about Volar [with this FREE video lesson from Vue School](https://vueschool.io/lessons/volar-the-official-language-feature-extension-for-vs-code?friend=vscode).
 
 ## IntelliSense
 
 As you start typing in `App.vue`, you'll see smart suggestions or completions both for HTML and CSS but also for Vue.js specific items like declarations (`v-bind`, `v-for`) in the Vue `template` section:
 
-![Vue.js suggestions](images/vuejs/suggestions.png)
-
-and Vue properties (`methods`, `computed`) in the `scripts` section:
-
-![Vue.js JavaScript suggestions](images/vuejs/javascript-suggestions.png)
+![vue directive suggestions](./images/vuejs/directive-suggestions.png)
 
-### Go to Definition, Peek definition
+and some Vue compiler micros in the script setup section:
 
-VS Code through the Vue extension language service can also provide type definition information in the editor through **Go to Definition** (`kb(editor.action.revealDefinition)`) or **Peek Definition** (`kb(editor.action.peekDefinition)`). Put the cursor over the `App`, right click and select **Peek Definition**. A [Peek window](/docs/editor/editingevolved.md#peek) will open showing the `App` definition from `App.js`.
-
-![Vue.js peek definition](images/vuejs/peek-definition.png)
-
-Press `kbstyle(Escape)` to close the Peek window.
+![vue macro suggestions](./images/vuejs/macro-suggestions.png)
 
 ## Hello World
 
 Let's update the sample application to "Hello World!". In `App.vue` replace the HelloWorld component `msg` custom attribute text with "Hello World!".
 
-```html
-<template>
-  <div id="app">
-    <img src="./assets/logo.png">
-    <HelloWorld msg="Hello World!"/>
-  </div>
-</template>
+```
+<HelloWorld msg="Hello World" />
 ```
 
-Once you save the `App.vue` file (`kb(workbench.action.files.save)`), restart the server with `npm run serve` and you'll see "Hello World!". Leave the server running while we go on to learn about Vue.js client side debugging.
```

Comment:
Please use the `kb(command ID)` convention. The VS Code website will then display the correct keybinding for the reader's platform. 
We shouldn't hard-code just the macOS keyboard shortcut.